docs/publishing.md

```diff
@@ -6,53 +6,49 @@ Lerna is a tool that optimizes the workflow around managing multi-package reposi
 
 It manages dependencies between our components so that if the version number of `globals` is bumped, all components dependent on `globals` will also have their version bumped.
 
-### Publishing components manually (while in Alpha)
+### Publishing components manually (while in Alpha/Private beta)
 
 In a new branch:
 
-1. Run the `npm run build:packages` task.
+1. Update `CHANGELOG` with the new version number and the changes made.
 
-2. Ensure all changes to `src/components/component-name` have been copied to `packages/component-name`.
+2. Run the `npm run build:packages` task.  
 
-3. For any new components, add a `package.json` inside `packages/component-name`:
+3. Ensure all changes to `src/components/component-name` have been copied to `packages/component-name`. In order to verify this, do the following in the Design System:
```

Comment:
I think this is worth doing, but doing this only verifies that the compiled packages _work_ - it _doesn't_ mean that all of the changes have been copied.

Reply:
Good point. I've reworded this to make it clearer.